examples/get_started/common_sql_functions.py

```diff
@@ -3,37 +3,40 @@
 from datachain.sql.functions import array, greatest, least, path, string
 
 
-def num_chars_udf(name):
-    parts = name.split(".")
+def num_chars_udf(file):
+    parts = file.name.split(".")
     if len(parts) > 1:
         return (list(parts[1]),)
     return ([],)
 
 
 ds = DataChain.from_storage("gs://datachain-demo/dogs-and-cats/")
-ds.map(num_chars_udf, params=["file.name"], output={"num_chars": list[str]}).select(
-    "file.name", "num_chars"
+ds.map(num_chars_udf, params=["file"], output={"num_chars": list[str]}).select(
+    "file.path", "num_chars"
 ).show(5)
 
 (
     ds.mutate(
-        length=string.length(C("file.name")),
-        parts=string.split(C("file.name"), literal(".")),
+        length=string.length(path.name(C("file.path"))),
+        parts=string.split(path.name(C("file.path")), literal(".")),
     )
-    .select("file.name", "length", "parts")
+    .select("file.path", "length", "parts")
     .show(5)
 )
 
 (
-    ds.mutate(stem=path.file_stem(C("file.name")), ext=path.file_ext(C("file.name")))
-    .select("file.name", "stem", "ext")
+    ds.mutate(
+        stem=path.file_stem(path.name(C("file.path"))),
+        ext=path.file_ext(path.name(C("file.path"))),
```

Comment:
Why can't it be simplified:
```python
        stem=path.file_stem(C("file.path")),
        ext=path.file_ext(C("file.path")),
```

Reply:
`stem` would look like `dogs-and-cats/cat.1` instead of the expected `cat.1`.

Reply:
that's seems like a bug in `file_stem()`

That's a regular functionality:
```python
>>> Path('dogs-and-cats/cat.1.jpg').stem
'cat.1'
```